package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1751,14 +1767,14 @@ def generate_table(self, output_format=None):
         """Generate a normalised table of the results.
 
```

Comment:
add explanations
```reST
Parameters
----------
output_format : {'sele1_sele2', 'donor_acceptor'}
     The output format of the `table` can be changed a fashion similar to
     :attr:`WaterBridgeAnalysis.results.timeseries` by changing the labels
     of the columns of the participating atoms.

Returns
-------
table : numpy.recarray
     A "tidy" table with one hydrogen bond per row, labeled according to 
    `output_format` and containing information of atom_1, atom_2, distance,
    and angle.
```

Reply:
If there's an explanation of the format of table then it could go here, at least in a short version.